general/package/datalink/datalink.mk

```diff
@@ -26,7 +26,7 @@ define DATALINK_INSTALL_TARGET_CMDS
 	cp $(DATALINK_PKGDIR)/files/$(DATALINK_FPATH)/telemetry $(TARGET_DIR)/usr/bin
 
 	$(INSTALL) -m 755 -d $(TARGET_DIR)/usr/sbin
-	cp $(DATALINK_PKGDIR)/files/$(DATALINK_FPATH)/channels.sh $(TARGET_DIR)/usr/sbin
+	cp $(DATALINK_PKGDIR)/files/$(DATALINK_FPATH)/channels.sh $(TARGET_DIR)/root
```

Comment:
In [mavfwd code](https://github.com/OpenIPC/mavfwd/commit/feac9dad090bca82e3b63de14174ca9aada0f8b1), channels.sh run without custom path. It can be located only in /bin /sbin /usr/bin or /usr/sbin